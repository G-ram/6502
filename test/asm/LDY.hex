a0 05 8c 00 01 
