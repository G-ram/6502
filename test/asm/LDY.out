A: 00
X: 00
Y: 05
P: 00010000
S: 00
PC: 605
100 : 05
101 : 00
